extension/chrome/settings/inbox/inbox.ts

```diff
@@ -345,6 +347,22 @@ Catch.try(async () => {
       for (const m of thread.messages) {
         await renderMsg(m);
       }
+
+      if (threadHasPGPMessage) {
+        $(".action_see_original_message").css('display', 'inline-block');
+        if (showOriginal) {
+          $(".action_see_original_message").text('See Decrypted');
+          $(".action_see_original_message").click(Ui.event.prevent('double', async (self) => {
+            loadUrl({ "acctEmail": acctEmail, "threadId": threadId });
```

Comment:
Should be shortened to (that's what the lint was probably complaining about)
```js
loadUrl({ acctEmail, threadId });
```
Since it's so short, might as well:
```js
$(".action_see_original_message").click(Ui.event.prevent('double', () => loadUrl({ acctEmail, threadId })));
```
The `double` is trying to prevent double clicks, but here it wouln't do any harm, so you can do:
```js
$(".action_see_original_message").click(Ui.event.handle(() => loadUrl({ acctEmail, threadId })));
```
Just to make sure errors are handled / reported.